MainWindow.xaml.cs

```diff
@@ -644,8 +642,9 @@ private void SetSupportedDriveSpeed()
                 return;
             }
 
-            // Validate that the required program exits
-            if (!File.Exists(dicPath))
+            // Validate that the required program exits and it's not DICUI itself
```

Comment:
Nit: I can't spell apparently... If you can change `program exits` to `program exists`, I'd appreciate it